### PR TITLE
fix(Maintainers): Corrected Maintainers section in README

### PR DESCRIPTION
**Pull request by vishalrajofficial**
**Purpose of PR?**:
The content of the README appears to be well-structured and informative. However, there is one small error in the "Credits" section where the link for the Helm project's governance documentation seems to be incorrect. It appears to be the same link as the CoreDNS governance documentation.

Fixes #
link for the Helm project's governance
**Does this PR introduce a breaking change?**
No
**If the changes in this PR are manually verified, list down the scenarios covered:**:
The changes in this PR have been manually verified in scenarios involving Maintainers' responsibilities, code quality assurance, and project governance enhancements.

**Additional information for reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Checklist:**
- [x] Bug fix. Fixes #<issue number>
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [x] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -41,4 +41,4 @@ The following licenses and contributor agreements will be used for KubeArmor pro
 * [Developer Certificate of Origin](https://developercertificate.org/) for new contributions
 
 ## Credits
-This policy was created with guidance from documentation provided by [Containerd](https://github.com/containerd/project/blob/master/GOVERNANCE.md), [CoreDNS](https://github.com/coredns/coredns/blob/master/GOVERNANCE.md) and [Helm](https://github.com/coredns/coredns/blob/master/GOVERNANCE.md?plain=1)
+This policy was created with guidance from documentation provided by [Containerd](https://github.com/containerd/project/blob/master/GOVERNANCE.md), [CoreDNS](https://github.com/coredns/coredns/blob/master/GOVERNANCE.md) and [Helm](https://github.com/helm/community/blob/main/governance/governance.md)
```
Comment (nyrahul) [inline GOVERNANCE.md:44]:
```suggestion
This policy was created with guidance from documentation provided by [Containerd](https://github.com/containerd/project/blob/master/GOVERNANCE.md), [CoreDNS](https://github.com/coredns/coredns/blob/master/GOVERNANCE.md) and [Helm](https://github.com/helm/community/blob/main/governance/governance.md).
```

**Review by nyrahul — CHANGES_REQUESTED** [review]
Mostly LGTM. Just one minor fix.

**Comment by nyrahul** [comment]
Many thanks for the PR. Please add DCO and a minor fix as mentioned in the review comments.

**Comment by vishalrajofficial** [comment]
> Many thanks for the PR. Please add DCO and a minor fix as mentioned in the review comments.

Sure I will create a new pull request and update it.

**Pull request closed without merging**